packages/xsschema/src/wrap.ts

```diff
@@ -0,0 +1,29 @@
+import type { StandardSchemaV1 } from '@standard-schema/spec'
+
+export const wrapArray = async <T extends StandardSchemaV1>(schema: T): never | Promise<T> => {
+  const vendor = schema['~standard'].vendor
+  try {
+    // eslint-disable-next-line ts/no-unsafe-assignment
+    const { array } = await import(vendor)
+    // eslint-disable-next-line ts/no-unsafe-return, ts/no-unsafe-call
+    return array(schema)
+  }
+  catch {
+    throw new Error(`xsschema: Unsupported schema vendor "${vendor}"`)
+  }
+}
+
+export const wrapObject = async <T extends StandardSchemaV1>(schema: T, name: string): never | Promise<T> => {
+  const vendor = schema['~standard'].vendor
+  try {
+    // eslint-disable-next-line ts/no-unsafe-assignment
+    const { object } = await import(vendor)
```

Comment:
We should consider vendors that do not export `object`

Reply:
maybe throw an error to notice user that vendor do export `object`?